### DISCUSSION TITLE
my tablet is supported but it is not even searched

### DISCUSSION THREAD

**Discussion by dekaqq** [OpenTabletDriver/OpenTabletDriver#3356] [Q&A] [answered]
I downloaded the driver from the flathub application because the version from github does not work and there was a problem
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/160600532/ab031786-c749-4e55-8db2-24739b817e1c)
the plugin doesn't even try to find my tablet, ctl 472, help me download an adequate version or fix something, if anything - ubuntu

**Comment by jamesbt365** [comment, marked as answer]
You cut off the error message but it is actually looking for your tablet. You were vague about the version on GitHub not working, but you really should use it explain what about it isn't working.

Most installations of OTD will not work properly until you reboot and may require a manual rebuild of your systemwide initramfs, but that is rare. If you can better explain what about it isn't working I'll be able to help you better.

You however should not use the Flatpak, it is *not* an official package and if you read the docs on the manifest you'll see that it requires manual intervention to setup, which is why it is not working in your case. Unless there is not an official package available you should not be using the Flatpak.

**Reply by dekaqq** [reply]
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/160600532/b88d8d47-b5ff-4af0-91c3-bfc288029390)
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/160600532/c7d3e9c9-4b3b-425d-b99a-34080e9ca1b3)
![image](https://github.com/OpenTabletDriver/OpenTabletDriver/assets/160600532/feec069b-fee4-4964-b44f-7f5f7ea82bda)

**Reply by jamesbt365** [reply]
You should not be using the dotnet install script to install dotnet, this is likely why you are unable to install dotnet with your package manager.

Uninstall dotnet using the script, remove the microsoft repo if you have it, then install OpenTabletDriver through the deb.

**Reply by dekaqq** [reply]
can u help me pls

**Reply by jamesbt365** [reply]
Reverse the dotnet install script and install it from your distros respositories, you should be able to figure out how to undo this online.

You have more information about your system than I do.
